deploy-manage/uninstall/delete-a-cloud-deployment.md

```diff
@@ -1,35 +1,93 @@
 ---
-mapped_urls:
-  - https://www.elastic.co/guide/en/cloud-heroku/current/ech-delete-deployment.html
-  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-delete-deployment.html
-  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-terminate-deployment.html
-  - https://www.elastic.co/guide/en/cloud/current/ec-delete-deployment.html
-  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-restore-deployment.html
-  - https://www.elastic.co/guide/en/cloud/current/ec-billing-stop.html
-  - https://www.elastic.co/guide/en/serverless/current/general-billing-stop-project.html
+applies_to:
+  deployment:
+    ess:
+    ece: 
+  serverless:
 ---
 
 # Delete a cloud deployment
```

Comment:
both "orchestrated" and "cloud" don't have really clear implications. here, I'd expect to see ECK as well. we can leave that alone for now, but can you create an issue for getting deletion of an ECK deployment into these docs?  for now, I suggest changing the title to state ECE, ECH, + serverless specifically, and avoid using the word "cloud"

Reply:
yeah, we'll just have to accept it for now. we've identified the issue and have a plan to fix :)